src/scss/components/banner/_banner-quote-text.scss

```diff
@@ -0,0 +1,86 @@
+@charset 'UTF-8';
+
+// Top banner styles
+.irsr-wrapper--banner-quote-text {
+  .su-hero {
+    display: block;
+    min-height: 220px;
+  
+    .su-hero__media {
+      min-height: auto;
+      position: absolute;
+  
+      img {
+        opacity: 0.4;
+      }
+    }
+  
+    .su-hero__card {
+      background: transparent;
+      border: 0;
+      box-shadow: none;
+      left: auto;
+      text-align: center;
+      max-width: unset;
+      margin: 0;
+      bottom: unset;
+  
+      .su-card__contents {
+        @include responsive-spacing('padding-top', 4);
+        @include responsive-spacing('padding-bottom', 5);
+
+        color: $su-color-white;
+        width: 75%;
+        margin: 0 auto;
+        min-height: 220px;
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+
+        @include grid-media-max('sm') {
+          width: 100%;
+        }
+
+        :first-child {
+          margin-top: 0;
+        }
+  
+        h2 {
+          @include type-a;
+          font-family: $su-font-serif;
+        }
+  
+        p,
+        blockquote {
+          @include big-paragraph;
+          margin-right: 0;
+          margin-left: 0;
+
+          &:first-of-type {
+            &::before {
+              display: block;
+              content: open-quote;
+              rotate: 180deg;
+              font-size: 8rem;
+              transform: scaleX(-3) scaleY(2.8);
+              height: auto;
+              position: relative;
+              bottom: 5rem;
+              font-family: 'Font Awesome\ 5 Free';
```

Comment:
I don't think any additional action is needed at the moment but we'll want to double check the following with the client
![image](https://github.com/SU-SWS/ppo_irsr_subtheme/assets/34019925/4959eca1-27a6-4ede-b245-080b521db10b)

- Is this style of open-quote okay or should we use the one shown in the mock?

![image](https://github.com/SU-SWS/ppo_irsr_subtheme/assets/34019925/64702747-4f1d-4098-ad88-68f33c9949cb)
- In the mock, there is this note associated with the open-quote. Should the editor be able to toggle the quote? If so, since this is a banner variant, I imagine that we'd provide two options (e.g. Banner Quote Text (with icon), Banner Quote Text (no icon) or something of the sort).

Reply:
Thanks @rebeccahongsf! I asked for clarification on the quote, and they are happy with the current implementation, so no change needed there. Great point about the no icon banner - I added that and named it the transparent overlay banner variant, since it can be used more broadly than just quotes.